packages/material-ui/src/NativeSelect/NativeSelectInput.js

```diff
@@ -88,7 +88,7 @@ export const nativeSelectIconStyles = ({ styleProps, theme }) => ({
     color: theme.palette.action.disabled,
   },
   ...(styleProps.open && {
-    right: 7,
+    transform: 'rotate(180deg)',
```

Comment:
For reference, the same on master - https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/NativeSelect/NativeSelect.js#L83